bloom/generators/debian/templates/rules.em

```diff
@@ -59,4 +60,7 @@ override_dh_auto_install:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_install
+	dh_auto_install --destdir=$(CURDIR)/debian/@(Package)
```

Comment:
This is needed because dh_auto_install installs into debian/tmp instead of debian/@(Package) when we introduce a 2nd debian package (the -dbg package) to the control file. Asking dh_auto_install to install the content explicitly into debian/@(Package) using --destdir is often used in such build configurations as an alternative to using .install files.